resources/charts/fork-observer/values.yaml

```diff
@@ -0,0 +1,133 @@
+# Default values for fork-observer.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+namespace: warnet
+
+restartPolicy: Always
+
+image:
+  repository: b10c/fork-observer
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: "latest"
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+podLabels:
+  app: "warnet"
+  mission: "observer"
+
+podSecurityContext: {}
+  # fsGroup: 2000
+
+securityContext: {}
+  # capabilities:
+  #   drop:
+  #   - ALL
+  # readOnlyRootFilesystem: true
+  # runAsNonRoot: true
+  # runAsUser: 1000
+
+service:
+  type: ClusterIP
+
+ingress:
+  enabled: false
+  className: ""
+  annotations: {}
+    # kubernetes.io/ingress.class: nginx
+    # kubernetes.io/tls-acme: "true"
+  hosts:
+    - host: chart-example.local
+      paths:
+        - path: /
+          pathType: ImplementationSpecific
+  tls: []
+  #  - secretName: chart-example-tls
+  #    hosts:
+  #      - chart-example.local
+
+resources: {}
+  # We usually recommend not to specify default resources and to leave this as a conscious
+  # choice for the user. This also increases chances charts run on environments with little
+  # resources, such as Minikube. If you do want to specify resources, uncomment the following
+  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+  # limits:
+  #   cpu: 100m
+  #   memory: 128Mi
+  # requests:
+  #   cpu: 100m
+  #   memory: 128Mi
+
+
+livenessProbe:
+  exec:
+    command:
+    - pidof
+    - fork-observer
+  failureThreshold: 3
+  initialDelaySeconds: 5
+  periodSeconds: 5
+  successThreshold: 1
+  timeoutSeconds: 1
+readinessProbe:
+  failureThreshold: 1
+  periodSeconds: 1
+  successThreshold: 1
+  tcpSocket:
+    port: 2323
+  timeoutSeconds: 1
+
+
+# Additional volumes on the output Deployment definition.
+volumes: []
+# - name: foo
+#   secret:
+#     secretName: mysecret
+#     optional: false
+
+# Additional volumeMounts on the output Deployment definition.
+volumeMounts: []
+# - name: foo
+#   mountPath: "/etc/foo"
+#   readOnly: true
+
+port: 2323
+
+# This should be copied to /app/config.toml
+baseConfig: |
+  # fork-observer base configuration file
+
+  # Database path of the key value store. Will be created if non-existing.
+  database_path = "db"
+
+  # path to the location of the static www files
+  www_path = "./www"
+
+  # Interval for checking for new blocks
+  query_interval = 10
+
+  # Webserver listen address
+  address = "0.0.0.0:2323"
+
+  # Custom footer for the site.
+  footer_html = """
+      <div class="my-2">
+        <div>
+          <span class="text-muted">Warnet fork-observer</span>
+        </div>
+      </div>
+    """
+
+  [[networks]]
+  id = 0xDEADBE
+  name = "Warnet"
+  description = "A Warnet"
+  min_fork_height = 0
+  max_interesting_heights = 5
```

Comment:
you probably don't need and want to set this. It will only show you the latest 5 forks

Reply:
I'm thinking we will want to move these values out into a config.yaml for fork-observer (same as we do for networks/namespaces) and allow these values to be set by the user. For some use cases, someone might only want to see max 5, but for the Tabconf use case we will want more.